tasks/docker_image_prune.yml

```diff
@@ -0,0 +1,40 @@
+# SPDX-FileCopyrightText: 2025 Stefan Haun <tux@netz39.de>
+# SPDX-License-Identifier: MIT
+---
+- block:
+    - name: Copy docker-image-prune.service file
+      ansible.builtin.copy:
+        src: files/docker-image-prune.service
+        dest: /etc/systemd/system/docker-image-prune.service
+        mode: "0644"
+      notify: Reload systemd
+
+    - name: Copy docker-image-prune.timer file
+      ansible.builtin.copy:
+        src: files/docker-image-prune.timer
+        dest: /etc/systemd/system/docker-image-prune.timer
+        mode: "0644"
+      notify: Reload systemd
+
+    - name: Enable and start docker-prune.timer
+      ansible.builtin.systemd:
+        name: docker-image-prune.timer
+        enabled: yes
+        state: started
+
+  when: docker_image_prune
+
+
+# Note: This check can go away in a few releases
```

Comment:
could we not remove the offending line automatically?

Reply:
I considered this, but it is inserted into the generic crontab and I did not want to mess with that. We do not really know what has been added by the user and what has been added by Ansible. Better not risk destroying anything.

It would have been  easier if the command would have gone to a distinct file, which we could then have deleted.